interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -422,6 +422,125 @@ def test_equil_results(gas, fuel, ox, Y_Cf, Y_Of, Y_Co, Y_Oo, basis):
 
         test_equil_results(gas, fuel, ox, Y_Cf, Y_Of, Y_Co, Y_Oo, 'mass')
 
+    def test_equivalence_ratio_simple_dilution(self):
+        gas = ct.Solution("gri30.yaml")
+
+        gas.X = "H2:4,O2:1,CO:3,CO2:4,N2:5,CH4:6"
```

Comment:
I think `phi = 2` should be defined outside of `test_simple_dilution` (right around where you define the mixture); it will be accessible without passing it as argument. Also, define `afr = 0.5 / phi` (or the reciprocal value) and use this to avoid the hard-coded factor '4'.